### PR TITLE
Trev8939/show labels on layers in 3d

### PR DESCRIPTION
**Pull request by TADraeseke**
Add's the now Show symbols on layers in 3D sample https://devtopia.esri.com/runtime/common-samples/blob/master/designs/ShowLabelsOnLayer3D/README.md

Can you review when you can please @Rachael-E? Thanks!

### CODE REVIEW

File: kotlin/show-labels-on-layer-in-3d/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Pan and zoom to explore the scene. Notice the labels showing installation dates of features in the 3D gas network.
+
+## How it works
+
+1. Create an `Scene` from a `PortalItem`.
+2. Add the scene to an `SceneView` and load it.
+3. After loading is complete, obtain the `FeatureLayer` from one of the `GroupLayer`s in the scene's `operationalLayers`.
+4. Create an `AGSTextSymbol` to use for displaying the label text.
+5. Create an `AGSLabelDefinition` using an  `AGSArcadeLabelExpression`.
```
Comment (Rachael-E) [inline kotlin/show-labels-on-layer-in-3d/README.md:21]: oops iOS classes have made their way in! 
```suggestion
4. Create an `TextSymbol` to use for displaying the label text.
5. Create an `LabelDefinition` using an  `ArcadeLabelExpression`.
```

**Review by Rachael-E — COMMENTED** [review]
thanks @TADraeseke ! just a few little things to fix up here, nothing major. Keen to hear your thoughts on the difference between the "keywords" section of the json file between Android and Java and which one we should be going for. 

Note I haven't run this app locally due to not having Android Studio set up yet as we discussed: so for now this is purely a code review :)

File: kotlin/show-labels-on-layer-in-3d/README.metadata.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+    "category": "Visualization",
+    "description": "Display custom labels in a 3D scene.",
+    "formal_name": "ShowLabelsOnLayerIn3D",
+    "ignore": false,
+    "images": [
+        "show-labels-on-layer-in-3d.png"
+    ],
+    "keywords": [
```
[the remark this answers is not in the document]
Comment (TADraeseke) [reply kotlin/show-labels-on-layer-in-3d/README.metadata.json:9]: Ooo this I don't know. On Android the metadata files are produced with this tool: https://github.com/Esri/arcgis-runtime-samples-android/tree/master/tools

**Comment by TADraeseke** [comment]
Thanks @Rachael-E - Made those fixes! Can you re-review when you have a chance?

**Review by Rachael-E — APPROVED** [review]
looks great, thanks @TADraeseke !

**Pull request merged**